### ISSUE TITLE
Ventoy @ macOS 11.1 Big Sur

### ISSUE THREAD

**Issue by DragonSerW** [ventoy/Ventoy#680]
How can I make it work?

**Comment by ventoy** [comment]
Actually, mac machine is not fully supported. 
Because I don't have any mac machine and never test Ventoy on any mac machine.

**Comment by DX37** [comment]
> 
> 
> How can I make it work?

Maybe install `homebrew` on macOS and try to compile it from source?

**Comment by DragonSerW** [comment]
I have homebrew installed.
How should I compile it?

**Comment by DX37** [comment]
> I have homebrew installed.
> How should I compile it?

https://github.com/ventoy/Ventoy/blob/master/DOC/BuildVentoyFromSource.txt

**Comment by wuJflower** [comment]
Did u compile it?

**Comment by carljmosca** [comment]
This seems like an interesting, very useful project - not had a chance to try it yet.  When I have time, I and can try on my mac.

Github actions supports macOS...at least it is listed; not actually done anything but Linux with Actions yet but I have found it to be very feature-rich.  https://github.com/features/actions

**Comment by TheAlienDrew** [comment]
> > How can I make it work?
> 
> Maybe install `homebrew` on macOS and try to compile it from source?

@DX37 even if it compiles, it will have issues since there is no support for creating and writing to ext[2/3/4]/xfs partitions on mac (only read-only implementations that can be installed from brew)

**Comment by topiga** [comment]
> > > How can I make it work?
> > 
> > 
> > Maybe install `homebrew` on macOS and try to compile it from source?
> 
> @DX37 even if it compiles, it will have issues since there is no support for creating and writing to ext[2/3/4]/xfs partitions on mac (only read-only implementations that can be installed from brew)

You can have some support with [MacFuse](https://osxfuse.github.io/), see their [EXT wiki](https://github.com/osxfuse/osxfuse/wiki/Ext) and their [XFS wiki](https://github.com/osxfuse/osxfuse/wiki/XFS) (not with much info)

**Comment by TheAlienDrew** [comment]
> > > > How can I make it work?
> > > 
> > > 
> > > Maybe install `homebrew` on macOS and try to compile it from source?
> > 
> > @DX37 even if it compiles, it will have issues since there is no support for creating and writing to ext[2/3/4]/xfs partitions on mac (only read-only implementations that can be installed from brew)
> 
> You can have some support with [MacFuse](https://osxfuse.github.io/), see their [EXT wiki](https://github.com/osxfuse/osxfuse/wiki/Ext) and their [XFS wiki](https://github.com/osxfuse/osxfuse/wiki/XFS) (not with much info)

Yes, but that's highly unstable write mode, plus each MacOS update mostly breaks the driver anyways.

**Comment by FranklinYu** [comment]
A workaround is to boot the ISO in a virtual machine (with VirtualBox or VMware Fusion), and pass USB drive to the VM. See https://github.com/ventoy/Ventoy/issues/35#issuecomment-846714041 by @TheAlienDrew.
